src/components/Modules/withModule.js

```diff
@@ -13,7 +13,8 @@ const findModule = (modules, moduleName) => {
   return (
     findModuleInArray(modules.app, moduleName) ||
     findModuleInArray(modules.plugin, moduleName) ||
-    findModuleInArray(modules.settings, moduleName)
+    findModuleInArray(modules.settings, moduleName) ||
+    findModuleInArray(modules.handler, moduleName)
```

Comment:
This function is starting to looks silly. Do we really need an explicit enumeration, in code, of all the supported module types? Surely we can just just walk through the keys of `modules`?

(Not that this is really the concern of the present PR. Just a matter of hygiene.)

Reply:
Yes this is definitely something we can clean up.